tests/test_main.py

```diff
@@ -142,13 +103,13 @@ def test_main_ws_failed_publish(pubsub, mTockenAuth, running, filterMessage, cre
     'POINT_TOPIC': 'POINT_TOPIC',
```

Comment:
We still have several references to this 'POINT_TOPIC', are they needed with the swap to raw topic?

Reply:
Yeah those are extra, removing them now.